src/main/java/org/xbill/DNS/TXTBase.java

```diff
@@ -82,7 +82,9 @@ protected String rrToString() {
         sb.append(" ");
       }
     }
-    return sb.toString();
+    String s = sb.toString();
+    // always return at least an empty quoted String
+    return "".equals(s) ? "\"\"" : s;
```

Comment:
I'd rather do an `if (strings.isEmpty()){ return "\"\""; }` at the beginning to avoid unnecessary objects (iterator, string builder, string).

Reply:
Sure, that looks better. Updating.